Octokit/Models/Request/BranchProtectionUpdate.cs

```diff
@@ -122,6 +122,7 @@ public BranchProtectionSettingsUpdate(BranchProtectionRequiredStatusChecksUpdate
         public BranchProtectionSettingsUpdate(BranchProtectionRequiredStatusChecksUpdate requiredStatusChecks,
                                               BranchProtectionRequiredReviewsUpdate requiredPullRequestReviews,
                                               BranchProtectionPushRestrictionsUpdate restrictions,
+                                              bool requiredSignatures,
```

Comment:
Would you mind fixing these warnings by adding appropriate an appropriate docs comment?

Reply:
Warnings?

Reply:
> Would you mind fixing these warnings by adding appropriate an appropriate docs comment?

I will insert a new line with a comment on ln. 115

Reply:
Good, thanks!